src/components/movieDetails/movieDetails.js

```diff
@@ -1,34 +1,33 @@
-import './movieDetails.css';
+import './movieDetails.scss';
 
-const MovieDetails = ({ isOpen, onClose, movie }) => {
-    if (!isOpen) return null;
+const MovieDetails = ({ movie, onCloseMovieDetails }) => {
     return (
-        <div className="modal">
-            <div className="modal-content">
-                <span className="close" onClick={onClose}>&times;</span>
-                <div className="movie-card">
-                    <div className="movie-poster">
-                        <img src={movie.poster_path} alt="Movie Poster" />
+        <div className="movie-details-container">
+            <div className="movie-card">
+                <div className="movie-poster-details">
+                    <img src={movie.poster_path} alt="Movie Poster" />
+                </div>
+                <div className="movie-info">
+                    <div className='title-rating-container'>
+                        <p className="movie-title-details">{movie.title.toUpperCase()}</p>
+                        <div className="circle"><span className="number">{movie.vote_average}</span></div>
+                    </div>
+                    <p><strong></strong>{movie.genres.map((genre) => {
+                        return <span key={genre} className="genre"> {genre}{movie.genres.indexOf(genre) < movie.genres.length - 1 ? "," : "."} </span>
```

Comment:
Move it to a separate function

Reply:
Moved to separate function `renderGenres`